src/SFA.DAS.EmployerIncentives.Web/Views/Payments/ListPayments.cshtml

```diff
@@ -9,6 +9,8 @@
     ViewData["ZenDeskLabels"] = "ei-payment-applications";
 }
 
+<a href="/@Model.BackLinkUrl" class="govuk-back-link">Back</a>
```

Comment:
Can this use asp-route attributes so that it's consistent across the app?

Reply:
When I do that it puts /Hub as the link.

Reply:
Working now